libs/website/api/src/lib/schemas/author.graphql

```diff
@@ -21,3 +21,20 @@ query Author($id: ID, $slug: Slug) {
     ...FullAuthor
   }
 }
+
+query AuthorList($filter: String, $cursor: ID, $take: Int, $skip: Int) {
```

Comment:
Oh just saw we do `$filter: String`, can we have the full filter list here?

Reply:
doesn't it require changes in the API? and then also in other parts like editor?

Reply:
Nah, this is just how you define it locally, below it's applied partially.

String -> AuthorFilter

and then below 

filter: {name: $filter} -> filter: $filter